### PR TITLE
Do not allow setting m.push_rules account data directly.

### PR DESCRIPTION
**Pull request by clokep**
Spec PR for https://github.com/matrix-org/matrix-spec-proposals/pull/4010

See https://pr1763--matrix-spec-previews.netlify.app/client-server-api/#server-behaviour-10

Preview: https://pr1763--matrix-spec-previews.netlify.app

### CODE REVIEW

File: content/client-server-api/modules/account_data.md [hunk 1]
```diff
@@ -28,4 +28,9 @@ These events can also be received in a `/events` response or in the
 
 Servers MUST reject clients from setting account data for event types
 that the server manages. Currently, this only includes
-[m.fully\_read](#mfully_read).
+[`m.fully_read`](#mfully_read) and [`m.push_rules`](#mfully_read).
+This applies to both global and room-specific account data.
```
Comment (clokep) [inline content/client-server-api/modules/account_data.md:32]: Should we clarify that this corresponds to the 405 error code?

Comment (turt2live) [inline content/client-server-api/modules/account_data.md:32]:
```suggestion
[`m.fully_read`](#mfully_read) and [`m.push_rules`](#mfully_read).
This applies to both global and room-specific account data.

{{% boxes/note %}}
{{% changed-in v="1.10" %}} `m.push_rules` was added to the rejection
list.
{{% /boxes/note %}}
```

File: changelogs/client_server/newsfragments/1763.clarification [hunk 2]
```diff
@@ -0,0 +1 @@
+Clarify that the `m.push_rules` account data type cannot be set using the `/account_data` API.
```
Comment (turt2live) [inline changelogs/client_server/newsfragments/1763.clarification:1]:
```suggestion
Clarify that the `m.push_rules` account data type cannot be set using the `/account_data` API, as per [MSC4010](https://github.com/matrix-org/matrix-spec-proposals/pull/4010).
```

File: content/client-server-api/modules/account_data.md (hunk 1 above)
Comment (turt2live) [reply content/client-server-api/modules/account_data.md:32]: I think so, yes.

File: content/client-server-api/modules/account_data.md [hunk 3]
```diff
@@ -28,4 +28,9 @@ These events can also be received in a `/events` response or in the
...
+
+Servers must allow clients to read the above event types as normal. Note that
+`m.fully_read` only exists for room-specific account data and `m.push_rules`
+only exists for global account data.
```
Comment (turt2live) [inline content/client-server-api/modules/account_data.md:36]: This is a bit confusing when paired with the line above about how *both* types of account data are banned. I think we can get away with links to the respective modules (as above).

```suggestion
Servers must allow clients to read the above event types as normal.
```

**Review by turt2live — COMMENTED** [review]
generally lgtm - thanks for writing it up! A few nitpicks and different word suggestions.

**Comment by clokep** [comment]
I think this is ready for another look!

**Review by turt2live — APPROVED** [review]
Thanks!

**Pull request merged**